### PR TITLE
Add launch config and fix npm warning

### PR DESCRIPTION
**Pull request by bkircher**
This PR does not add any user-visible features. It just adds

- a `launch.json` to easily test the extension during development
- add `"licence"` to `package.json` to silence a complaint from npm

### CODE REVIEW

File: .vscodeignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
-.vscode/**
```
Comment (qinezh) [inline .vscodeignore]: The files under `.vscode` are not needed at runtime, we should ignore them, see https://code.visualstudio.com/api/working-with-extensions/publishing-extension#.vscodeignore

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,4 @@
 out/
 node_modules/
-.vscode/
-.vscode-test/
\ No newline at end of file
+.vscode-test/
+package-lock.json
```
Comment (qinezh) [inline .gitignore]: package-lock.json is intended to be checked into source control as far as I know

**Comment by qinezh** [comment]
@bkircher Thanks for your contribution, I added some comments, please take a look : )

**Comment by bkircher** [comment]
Both comments are absolutely correct. I added another commit to this PR that addresses both. You can squash them if you decide to merge this PR.

**Comment by qinezh** [comment]
LGTM, thanks @bkircher

**Pull request merged**